docs/psf_spec/high_level_overview.rst

```diff
@@ -0,0 +1,244 @@
+product on which all subsequent methodology is applied. The image must be a two-dimensional
+array of ``x`` and ``y`` coordinates with ``flux`` represented by array values. If a
+WCS -- contained within a FITS file, for instance -- is passed with the two-dimensional
+array then the position of sources can be given in sky coordinates, instead of pixel values.
+The given image is passed as an input to the chosen PSF fitting class.
+
+
+.. _Finder:
+
+Finder
+^^^^^^
+
+The finder is the first step in the PSF fitting process, as sources must be discovered in
+the image before any kind of fit can be applied to them. These accept the input ``image``
+and produce a `~astropy.table.Table` of detected sources, by a set of criteria internal to
+the given ``Finder`` (see, e.g., `~photutils.detection.DAOStarFinder` using ``roundness``
+and ``sharpness`` to determine if sources are point-like). If an initial set of detected
+sources is passed to the fitter as ``init_guesses`` then ``Finder`` is not run on the first
+pass of an iterative fitting class, instead using the provided positions and fluxes. See
+the :doc:`finder <finder>` documentation for more details.
+
+
+.. _Group Maker:
+
+Group Maker
+^^^^^^^^^^^
+
+The second block to run in the fitting process, the various group maker processes, such as
+`~photutils.psf.DAOGroup`, allow for the merging of astrometrically overlapping sources
+into a set of multiple stars. These stars must be fit simultaneously (see Fitter_) as a
+composite model, avoiding the potential issue where blended sources return incorrect fluxes,
+as affects aperture photometry. The block accepts the `~astropy.table.Table` output from
+Finder_ and returns a second `~astropy.table.Table` containing the same columns as the input
+as well as an additional column indicating the group number of each source. More information
+on this block can be found in :doc:`Group Maker <group_maker>`.
+
+
+.. _Fitter:
+
+Fitter
+^^^^^^
+Once a set of sources has been detected and grouped, each source must have its respective
+properties -- position and flux primarily -- determined. For this a ``Fitter`` instance is
+required; these are drawn from a separate class or instance of minimization routines, such as
+those given under the `~astropy.modeling` umbrella. The fitter must be a two-dimensional fitter,
+by nature of the image to be fit, accepted as an input to the instance. The class must then
+return the position of the source (or sources, if a composite model was supplied; see
+`Group Maker`_) and its total flux, as determined by the PSF model describing the telescope
+system -- optics, CCD, etc. -- that created the image. See `below <PSF Model_>`_ for more
+details on the PSF model, and the :doc:`fitter <fitter>` documentation for further
+information on the ``Fitter`` instance and its properties.
+
+
+.. _PSF Model:
+
+PSF Model
+^^^^^^^^^
+Perhaps the most important aspect of PSF fitting, the ``PSF Model`` describes the distribution
```

Comment:
```suggestion
 Perhaps the most variable aspect of PSF fitting, the ``PSF Model`` describes the distribution
```
?

Reply:
I was trying to lead into this section with the idea that while the rest of the sections (getting good sources, grouping blended things, etc.) are important, *this* section has to be done right or there's no point doing any of the other stuff at all... So important is what I meant (instead of variable, as in "most likely to change for each user"?), but this is slightly qualitative and perhaps subjective language; none of the other sections have lead-in.

I think the question is more:
do we just start this section with ```The ``PSF model`` describes the distribution```?